src/app/components/client/toolbar/AnnouncementDialog.module.scss

```diff
@@ -67,7 +71,10 @@
     margin: 0 auto;
 
     button {
-      all: unset;
+      background: none;
+      border: none;
+      margin: 0;
+      font: inherit;
```

Comment:
Assuming there isn't a better way to re-use in our CSS setup, you might consider defining e.g. `@mixin button-reset` so you can just `@include button-reset;` and avoid the duplication/make sure nothing is missed when resetting button styles.